scripts/package/build.sh

```diff
@@ -1,11 +1,16 @@
 #!/bin/bash -e
 
 NODE_ENV="${NODE_ENV:-production}"
+SRC_DIR="src"
+ES_DIR="dist/es"
 
 # es modules
 if [ "$TARGET" != 'site' ]; then
-  NODE_ENV=$NODE_ENV babel src --out-dir dist/es --ignore *.spec.js,__demo__ --source-maps --minified
+  NODE_ENV=$NODE_ENV babel "$SRC_DIR" --out-dir "$ES_DIR" --ignore *.spec.js,__demo__ --source-maps --minified
 fi
 
 # umd
 TARGET=$TARGET NODE_ENV=$NODE_ENV webpack
+
+# flow definitions
+flow-copy-source -v -i '**/__tests__/**' -i '**/__demo__/**' "$SRC_DIR" "$ES_DIR"
```

Comment:
The site package should not presently need to copy flow definitions to dist. See https://github.com/mineral-ui/mineral-ui/issues/24 for more context and https://github.com/mineral-ui/mineral-ui/pull/25 for implementation.